packages/esm-patient-list-management-app/src/lists-table/lists-table.component.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { type CSSProperties, useCallback, useId, useMemo, useState, useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
+import fuzzy from 'fuzzy';
+import orderBy from 'lodash-es/orderBy';
```

Comment:
The linter rule that requires this strikes me as wrong. It's correct that, for tree-shaking, you should import the function from `lodash`, but the idea of `lodash-es` is that its standard ES module, so `import { orderBy } from 'lodash-es'` should be fully tree-shakeable. Maybe I've misunderstood `lodash-es` though?

Reply:
Not sure, I'd have to look into that. It's entirely possible that the import syntax you've suggested is fully tree-shakeable, which would mean we would need to change the rule.